grails-app/services/au/org/ala/profile/hub/ExportService.groovy

```diff
@@ -126,25 +123,140 @@ class ExportService {
             withPool(THREAD_COUNT) {
                 children.eachParallel {
                     if (it.profileId != params.profileId && it.scientificName != params.profileId) {
-                        model.profiles << loadProfileData(it.profileId, model.opus, params)
+                        curatedModel.profiles << loadProfileData(it.profileId, opus, params, latest)
                     }
                 }
             }
 
-            model.profiles = model.profiles.sort { it.profile.scientificName }
+            curatedModel.profiles = curatedModel.profiles.sort { it.profile.scientificName }
         }
 
-        wkhtmltoxService.makePdf(
-                view: "/pdf/_profile",
-//                footer: "/pdf/_pdfFooter",
-//                header: "/pdf/_pdfHeader",
-                model: model,
-                marginLeft: 15,
-                marginTop: 20,
-                marginBottom: 20,
-                marginRight: 15,
-                headerSpacing: 10,
-        )
+        curatedModel.options << [
+                allowFineGrainedAttribution: opus.allowFineGrainedAttribution,
+                displayToc: curatedModel.profiles?.size() > 1
+        ]
+
+        curatedModel << [
+                cover: [
+                        title       : opus.title,
+                        subtitle    : curatedModel.profiles[0]?.profile?.fullName,
+                        logo        : opus.logoUrl,
+                        banner      : opus.bannerUrl,
+                        primaryImage: curatedModel.profiles[0]?.profile?.primaryImage?: (curatedModel.profiles[0]?.profile?.images?.size() > 0 ? curatedModel.profiles[0]?.profile?.images[0].leftImage.largeImageUrl : '')
+                ],
+                colophon: [
+                        copyright: opus.copyrightText,
+                        logo        : opus.logoUrl,
+                        profileLink: "${grailsApplication.config.grails.serverURL}/opus/${params.opusId}/profile/${curatedModel.profiles[0]?.profile?.uuid}"
+                ]
+        ]
+
+        return curatedModel
+    }
+
+    /**
+     *
+     * @param profileId
+     * @param opus
+     * @param params
+     * @param latest
+     * @return
+     */
+    private Map loadProfileData(String profileId, opus, Map params, boolean latest = false) {
+
+        Map model = [:]
+        model.profile = webService.get("${grailsApplication.config.profile.service.url}/opus/${opus.uuid}/profile/${URLEncoder.encode(profileId, "UTF-8")}?latest=${false}")?.resp
+
+        if (params.taxonomy || params.conservation) {
+            model.profile.speciesProfile = profileService.getSpeciesProfile(model.profile.guid)?.resp
+
+            model.profile.speciesProfile?.conservationStatuses?.each {
```

Comment:
We have overlooked the new status list info that Temi added. We'll need to include this in the PDF, too.